demisto_sdk/commands/common/tools.py

```diff
@@ -470,7 +470,10 @@ def get_last_remote_release_version():
     return ''
 
 
-def get_file(file_path, type_of_file):
+@lru_cache()
```

Comment:
1. Have you tried changing the max size? What is our cache miss ratio?
2. Would love to discuss the cache invalidation strategy